code/drasil-printers/lib/Language/Drasil/Markdown/Config.hs

```diff
@@ -0,0 +1,54 @@
+-- | Defines functions to create .toml and .csv config files for mdBook.
+module Language.Drasil.Markdown.Config where
+
+import Text.PrettyPrint (Doc, text, vcat, (<+>))
+import Data.Map (empty, elems)
+import Control.Lens
+import System.FilePath (takeFileName)
+
+import Language.Drasil.Markdown.Print (pSpec)
+import Language.Drasil.Printing.PrintingInformation (PrintingInformation(..))
+import Database.Drasil (labelledcontentTable)
+import Language.Drasil.Printing.Import.Sentence (spec)
+import Language.Drasil.Printing.LayoutObj (Filepath)
+
+import Language.Drasil hiding (Expr)
+
+-- | Generates the .toml config file for mdBook.
+makeBook :: Document -> PrintingInformation -> Doc  
+makeBook (Document t _ _ _) sm = vcat [
+  text "[book]",
+  text "language = \"en\"",
+  text "multilingual = false",
+  text "src = \"src\"",
+  text "title =" <+> mkTitle sm t,
+  text "[output.html]",
+  text "smart-punctuation = true",
+  text "mathjax-support = true"
+  ]
+makeBook _ _ = error "Type not supported: Notebook."
+
+-- | Generates the .csv file mapping the original 
+-- filepaths of assets to the location mdBook uses.
+makeCSV :: PrintingInformation -> Doc
+makeCSV sm = vcat $ map line assets
+  where
+    line (a, b) = text $ a ++ "," ++ b
+    assets = assetMap sm
```

Comment:
1. I think we can move a generic "make CSV" function to `drasil-utils` that works on purely String matrices.
2. Making a CSV is a bit more complicated than concatenation with a comma -- you need to escape some characters and use double quotation marks when appropriate.
3. I like it when CSVs have header rows. Do you think we can add them?

Reply:
Added in https://github.com/JacquesCarette/Drasil/pull/3848/commits/d880a81de3b040ee2ac707e358cc3ea209b8904a. I made a new module as I couldn't find an existing one where this would fit.

- I add double quotes around items that contain a `,`, `"`, `\n`, or an empty space.
- I escape double quote characters `"`, using the CSV escape char, which by default is a double quote. 
- Added headings to the CSV file that generates with the mdBook projects.

Please let me know of any other formatting details for CSV files if I missed any.